processor/src/main/java/com/ihren/processor/sink/PersonSink.java

```diff
@@ -0,0 +1,11 @@
+package com.ihren.processor.sink;
+
+import com.ihren.model.Person;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PersonSink {
+    public Person sink(Person person) {
```

Comment:
It is not a sink. Sink mens that you have some output and you produce it somewhere.
In this pipeline you have configured Function with Source and Sink in configuration what mean that you don't need to have it.
So, it is just mapper.